### PR TITLE
[11.0][IMP] mdc: manual worksheet open & close flag

### PR DESCRIPTION
**Pull request by dalonsod**
This improvement create new flags for a worksheet, which are filled when a massive open or close worksheets action is fired. With these flags it's possible to determine whether a worksheet was manually created, instead of a physical one

### CODE REVIEW

File: mdc/views/data_views.xml [hunk 1]
```diff
@@ -74,10 +79,12 @@
                         <group>
                             <field name="physical_open" readonly="True"/>
                             <field name="physical_start_datetime" readonly="True"/>
+                            <field name="manual_open" readonly="True"/>
```
Comment (lmiguens-solvos) [inline mdc/views/data_views.xml]: I think it´s better in "gr_worksheet_other_data" group and if it was editable

File: mdc/views/data_views.xml [hunk 2]
```diff
@@ -74,10 +79,12 @@
...
                         </group>
                         <group>
                             <field name="physical_close" readonly="True"/>
                             <field name="physical_end_datetime" readonly="True"/>
+                            <field name="manual_close" readonly="True"/>
```
Comment (lmiguens-solvos) [inline mdc/views/data_views.xml]: I think it´s better in "gr_worksheet_other_data" group and if it was editable when end_datetime be covered
<field name="manual_close" attrs="{'readonly': [('end_datetime', '=', False)]}"/>

File: mdc/models/morders.py [hunk 3]
```diff
@@ -563,6 +563,10 @@ def _default_date(self):
         default = _default_date)
     end_datetime = fields.Datetime(
         'End Datetime')
+    manual_open = fields.Boolean(
```
Comment (lmiguens-solvos) [inline mdc/models/morders.py:566]: In refactoring function in this file (line 936) need adds then new field physical_open to preserve this data.
,'physical_open': ws.physical_open

Comment (lmiguens-solvos) [reply mdc/models/morders.py:566]: ups! I was wrong. In line 936 you need to add the new field manual_open to perserve this data: ,'manual_open': ws.manual_open

File: mdc/views/data_views.xml (hunk 1 above)
Comment (dalonsod) [reply mdc/views/data_views.xml]: Done!

File: mdc/views/data_views.xml (hunk 2 above)
Comment (dalonsod) [reply mdc/views/data_views.xml]: Done!

File: mdc/models/morders.py (hunk 3 above)
Comment (dalonsod) [reply mdc/models/morders.py:566]: Done!

**Comment by dalonsod** [comment]
@lmiguens-solvos comments attended:
- now it's possible to (un)set manual flags from `mdc.worksheet` form.
- preserved `manual_open` when refactoring worksheets.
- added flag manual vs physical state validation.

**Review by lmiguens-solvos — APPROVED** [review]
LGTM 👍

**Pull request merged**